doc/user/pbr.rst

```diff
@@ -39,9 +39,9 @@ listing of ECMP nexthops used to forward packets for when a pbr-map is matched.
    sub-mode where you can specify individual nexthops.  To exit this mode type
    exit or end as per normal conventions for leaving a sub-mode.
 
-.. clicmd:: nexthop [A.B.C.D|X:X::X:XX] [interface] [nexthop-vrf NAME]
+.. clicmd:: nexthop {<A.B.C.D|X:X::X:XX>|interface} [nexthop-vrf NAME]
```

Comment:
I would hope we can still specify a interface with a v4 or v6 address?

Reply:
You still can. The idea of using curly braces is that you can specify a nexthop interface, a nexthop address or both (in any order).